datasets/census_bureau_international/_images/run_csv_transform_kub/csv_transform.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import glob
+import json
+import logging
+import os
+import pathlib
+import subprocess
+import typing
+
+import pandas as pd
+from google.cloud import storage
+
+
+def main(
+    source_url: typing.List[str],
+    source_file: typing.List[pathlib.Path],
+    target_file: pathlib.Path,
+    target_gcs_bucket: str,
+    target_gcs_path: str,
+    headers: typing.List[str],
+    pipeline_name: str,
+) -> None:
+    logging.info("Creating 'files' folder")
+    pathlib.Path("./files").mkdir(parents=True, exist_ok=True)
+
+    logging.info(f"Downloading file {source_url}")
+    download_file(source_url, source_file)
+
+    logging.info(f"Reading the file {source_url}")
+    if pipeline_name == "country_names_area":
+        df = pd.read_csv(source_file[0])
+    else:
+        df1 = pd.read_csv(source_file[0])
+        df2 = pd.read_csv(source_file[1])
+        df = pd.merge(df1, df2, how="left", on=["country_code"])
```

Comment:
You can use below code : 

```suggestion
    def read_files(source_files, joining_key):
    df = pd.DataFrame()
    for source_file in source_files:
        _df = pd.read_csv(source_file)
        if df.empty:
            df = _df
        else:
            df = pd.merge(df, _df, how="left", on=["country_code"])
    return df
```